contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -1,4 +1,138 @@
+
+For the Super-project, the *develop* and *master* branches are independent. Both branches track the latest changes in the corresponding library branch, for all libraries. The Super-project branches are never merged, as this strategy prevents merging. In other words, the Super-project *master* branch is created from the library *master* branches, not from the Super-project *develop* branch (which is primarily used for testing).
+
+When there is a public release of Boost, it is built from the *master* branch after that branch has been closed to updates (refer to the xref:release-process.adoc[]).
+
+=== Working with the Super-project
+
+You might consider creating submodules for a project, when developing your library. This is not a required process, but it does fit with some of the Super-project tooling:
+
+* In Git, when you add a submodule to a project, you make it point to a particular commit. This means that, when developers update their sub-projects, the Super-project _doesn't_ get immediately updated. For this reason, there is a commit bot that runs nightly and updates the commit to which each submodule references in the Super-project. This happens nightly and for both the *develop* and *master* branches. 
+
+* The implications of merging something in your library or submodule to the Super-project *develop* branch, include that due to how Boost is tested, you may affect other developers. As part of Boost testing, Continuous Integration (CI) usually clones the Super-project and any required submodules. 
+
+** When testing any branch that is _not_ *master*, the *develop* branch of the Super-project is cloned. 
+** When you're testing *master* or a release, the *master* branch of the Super-project is cloned. 
+
+While this cloning process is not a _requirement_, most libraries follow it.
+
+Merging something to develop may affect any dependent code in the rest of Boost as soon as the commit bot makes the update to the Super-project. For example, if a mistake is made to the MySQL library *develop* branch, then it will affect the CMake module, because this module runs tests to verify that you can use MySQL via CMake.
+
+NOTE:: Developers don't directly create Tags under repositories, when working with Boost. Tags are created by the release scripts, both in the Super-project and the individual repos, so do not add any of your own. This differs from the usual workflow in other non-Boost projects.
++
+If you need to look at the code for the 1.82.0 release, navigate to the relevant repo, and enter:
++
+[source,text]
+----
+git checkout boost-1.82.0 
+----
+
+== Breaking Changes
+
+Boost maintains its release versions with https://github.com/boostorg/boost/tags[Tags], where the tag is always the `boost-` prefix followed by a https://semver.org/[Semantic Versioning] string.  Using semver strings is useful because that's something most tools understand, and for most users who just want to find a certain boost version, that's enough information.
```

Comment:
I think we've got a little bit of a contradiction in this section - we're essentially saying "we use semantic versioning, except that we don't use semantic versioning at all"

Reply:
Agreed, reworded.